src/main/java/org/stellar/sdk/responses/AccountResponse.java

```diff
@@ -393,7 +393,7 @@ public String get(String key) {
     @SerializedName("transactions")
     private final Link transactions;
 
-    Links(Link effects, Link offers, Link operations, Link self, Link transactions) {
+    public Links(Link effects, Link offers, Link operations, Link self, Link transactions) {
```

Comment:
@lijamie98 , was just curious what is the use case driving the need to populate the model manually rather than from server api responses?

Reply:
Here is where I use it
https://github.com/stellar/java-stellar-anchor-dev/pull/7/files#diff-bb19215491831165a984cf8e66a77832f8567401759bba44621d8a21ff46dbceR169

This is an work around. I had to create another class TestSigner to convert to Signer at:
https://github.com/stellar/java-stellar-anchor-dev/pull/7/files#diff-bb19215491831165a984cf8e66a77832f8567401759bba44621d8a21ff46dbceR42

Reply:
It is actually the Signer class that I need a public constructor. However, similar pattern may occur when we do tests in the future. 

Builder pattern would work too if public constructors are not preferred. Changing to Builder pattern will also need to change the way we construct AccountResponse.

Reply:
I see, it's for testing concerns. I noticed a few other response POJO's here in the SDK already had public constructors on their inner classes, so, hopefully this is contained to AccountResponse. I wanted to also mention, Spring's `ReflectionTestUtils` , as an optional brute force way for tests to access private members of classes under test. I didn't see Spring in the anchor project's pom.xml, but maybe could use that with test scope if this issue of access crops up more.

Yes, It might be nice to for us to consider adding Lombok here into the SDK on these POJOs, and bring in builders. I see you have that in the anchor project as well, nice!

Reply:
One of the anchor dev project requirements is NOT to use Spring :-(. So, I used gson to bruteforce it. However, I think this can be better solved with either a builder or public constructor. 

Lombok is a real nice way to reduce the verboseness of Java. :-)  +1

I will change the CHANGELOG on Monday and update the PR.